light-clients/ics10-grandpa-cw/src/contract.rs

```diff
@@ -305,10 +299,29 @@ fn process_message(
 }
 
 #[cfg_attr(not(feature = "library"), entry_point)]
-pub fn query(_deps: Deps, _env: Env, msg: QueryMsg) -> StdResult<Binary> {
+pub fn query(deps: Deps, _env: Env, msg: QueryMsg) -> StdResult<Binary> {
+	let client_id = ClientId::from_str("08-wasm-0").expect("client id is valid");
 	match msg {
 		QueryMsg::ClientTypeMsg(_) => unimplemented!("ClientTypeMsg"),
 		QueryMsg::GetLatestHeightsMsg(_) => unimplemented!("GetLatestHeightsMsg"),
+		QueryMsg::Status(StatusMsg {}) => {
+			let client_state = get_client_state::<HostFunctions>(deps).unwrap();
```

Comment:
Can we handle the error?

Reply:
good call, added along with a unit test for it in the wasm light client